### PR TITLE
Fix project_sketch plot

### PR DESCRIPTION
**Pull request by wangliang1989**

### CODE REVIEW

File: source/scripts/project_sketch.sh [hunk 1]
```diff
@@ -17,6 +17,11 @@ EOF
 gmt begin project_sketch png,pdf
 
 p_az=$((270 - $profile_az))
+# 绘制原始坐标系
+gmt basemap -R0/10/0/10 -Jx1c -Bws --MAP_FRAME_TYPE=graph -p180/90
```
Comment (seisman) [inline source/scripts/project_sketch.sh]: `-p180/90` is not needed here.

File: source/scripts/project_sketch.sh [hunk 2]
```diff
@@ -48,13 +53,12 @@ gmt plot -W1p,black -Sqn1:+Lh+f15p,7,blue+n0c/-0.5c -p << EOF
 0 0
 $p 0
 EOF
+# 绘制测线的起点和终点
 echo 0 0 | gmt plot -Sc0.2c -W1p,darkblue -Gdodgerblue -N -p
 echo $profile_length 0 | gmt plot -Sc0.2c -W1p,darkblue -Ggreen -N -p
 
-# 绘制原始坐标系
+# 回到原始坐标系
 gmt basemap -R0/10/0/10 -Jx1c -Bws --MAP_FRAME_TYPE=graph -p180/90
```
Comment (seisman) [inline source/scripts/project_sketch.sh]: I think this line is not necessary.
```
gmt basemap -R0/10/0/10 -Jx1c -Bws --MAP_FRAME_TYPE=graph -p180/90
```

Comment (wangliang1989) [reply source/scripts/project_sketch.sh]: 如果没有，basemap就还是测线坐标系

Comment (seisman) [reply source/scripts/project_sketch.sh]: 那应该用 
```
gmt plot -R0/10/0/10 -Jx1c -T 
```
你现在相当于原始坐标系画了两遍

Comment (seisman) [reply source/scripts/project_sketch.sh]: 我之前的脚本就是在尽量避免在不同坐标系之间切换

**Pull request merged**